app/Http/Controllers/AppRegisterController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\AppRegister;
+use App\Mail\InAppRegisterEmailVerify;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Mail;
+
+class AppRegisterController extends Controller
+{
+    public function index(Request $request)
+    {
+        abort_unless(config('auth.iar') == true, 404);
```

Comment:
(See comment in config/auth.php first)

Could we simplify this like below if we type-cast the env key/config to boolean - it would return more clearly / naturally imo

```suggestion
        abort_unless(config('auth.iar'), 404);
```